changelog/8000.improvment.md

```diff
@@ -0,0 +1,5 @@
+Remove console logging of conversation level F1-score and precision since these calculations were not meaningful.
+
+Add conversation level accuracy to core policy results logged to file in `story_report.json` after running `rasa test core` or `rasa test`.
+
+Add test to check that `story_report.json` contents are correct.
```

Comment:
I think you can scrap this line about adding a test - just to keep it more concise.

Also, typo in the filename improv**e**ment. I'm not sure, but I think this affects towncrier's behaviour, and it won't work properly if the file isn't named correctly